book/security.rst

```diff
@@ -929,10 +929,10 @@ other users. Also, as the admin user, you yourself want to be able to edit
 
 To accomplish this you have 2 options:
 
-* :doc:`Voters </cookbook/security/voters_data_permission>` allow you to
-  use business logic (e.g. the user can edit this post because they were
-  the creator) to determine access. You'll probably want this option - it's
-  flexible enough to solve the above situation.
+* :doc:`Voters </cookbook/security/voters>` allow you to use business logic
```

Comment:
`[...] to use business logic [...]` looks very generic to me. I'd like something like: `... apply your own business logic ...`, `... define the custom logic applied to ...`, etc.